keyboards/foxlab/orange75/32a/config.h

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2019 MechMerlin
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define VENDOR_ID    0xFEEB
+#define PRODUCT_ID   0x6060
+#define DEVICE_VER   0x0001
+#define MANUFACTURER Fox-Lab
+#define PRODUCT      Orange75
+#define DESCRIPTION  A custom 75% keyboard
+
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 15
+
+//                         0   1   2   3   4   5   6   7   8   9   A   B   C   D  E
+#define MATRIX_ROW_PINS { B5, B0, B1, B2, B3, B4 }
+#define MATRIX_COL_PINS { C2, C4, C5, C6, C7, A7, A6, A5, A4, A3, A2, A1, A0, C3, D7 }
+// #define MATRIX_COL_PINS { A0, A1, A2, A3, A4, A5, A6, A7, C7, C6, C5, C4, C3, C2, C1, C0, D7 }
+#define UNUSED_PINS {}
+
+#define DIODE_DIRECTION COL2ROW
+#define DEBOUNCE 5
+
+#define BACKLIGHT_LEVELS 1
```

Comment:
So I watched the porting stream for this board 😁 you know how you were trying to find the pins for the indicator LEDs? May want to bookmark this: https://github.com/showjean/ps2avrU/blob/master/firmware/ps2avrGB/Circuit/ps2avrGB/20140829_140848_001.png
A lot of these BMC ports seem to just treat them as on/off backlight pins, eg. `backlight_set()` in winkeyless/bface, which is incorrect.

And I can see this PCB has backlight pads, so per the schematic that'll be:

```suggestion
#define BACKLIGHT_PIN D4
#define BACKLIGHT_LEVELS 3
```
D4 is a hardware PWM pin on the 32A, too 👍